### PR TITLE
LO28-30

### PR DESCRIPTION
**Pull request by jonthegeek**

### CODE REVIEW

File: 28-graphics_for_communication.Rmd [hunk 1]
```diff
@@ -2,7 +2,12 @@
 
 **Learning objectives:**
 
-- Jon will try to prefill these ahead of each week.
+- **Describe the contents** of a plot with **labels.**
+- Call out **specific features** of your data wition **annotations.**
```
Comment (jonathanbratt) [inline 28-graphics_for_communication.Rmd:6]:
```suggestion
- Call out **specific features** of your data with **annotations.**
```

File: 29-r_markdown_formats.Rmd [hunk 2]
```diff
@@ -2,7 +2,14 @@
 
 **Learning objectives:**
 
-- Jon will try to prefill these ahead of each week.
+- **Control R Markdown output** with **output formats** and **options.**
+- Recognize the available R Markdown **document formats.**
```
Comment (jonathanbratt) [inline 29-r_markdown_formats.Rmd:6]: "recognize" always gives me pause. I feel like it's a weaker form of "list" or "identify", which ...are more like questions than LOs? Not sure what to suggest, though.

**Review by jonathanbratt — CHANGES_REQUESTED** [review]
typo :)

File: 29-r_markdown_formats.Rmd (hunk 2 above)
Comment (jonthegeek) [reply 29-r_markdown_formats.Rmd:6]: I use it when you're supposed to read a thing and be able to maybe answer some multiple choice questions about it, but you aren't really expected to *know* the thing. I'm sure there's a better way to say it but I think this is good enough 🙃

**Review by jonathanbratt — APPROVED** [review]

**Pull request merged**